AppAuth-JS/built/xhr.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork